src/NJsonSchema/Validation/JsonSchemaValidator.cs

```diff
@@ -50,9 +50,10 @@ public JsonSchemaValidator(params IFormatValidator[] customValidators)
         /// <summary>Validates the given JSON data.</summary>
         /// <param name="jsonData">The json data.</param>
         /// <param name="schema">The schema.</param>
+        /// <param name="schemaType">The type of the schema.</param>
         /// <exception cref="JsonReaderException">Could not deserialize the JSON data.</exception>
         /// <returns>The list of validation errors.</returns>
-        public ICollection<ValidationError> Validate(string jsonData, JsonSchema schema)
+        public ICollection<ValidationError> Validate(string jsonData, JsonSchema schema, SchemaType schemaType = SchemaType.OpenApi3)
```

Comment:
I'd say the default should be JsonSchema and not OpenApi3 as this is a JSON Schema library first.

Reply:
Done